src/views/Create.vue

```diff
@@ -11,9 +11,25 @@
         </router-link>
       </div>
       <Block v-if="space.filters?.onlyMembers && !isMember">
-        <Icon name="warning" class="mr-1"/>
+        <Icon name="warning" class="mr-1" />
         {{ $t('create.onlyMembersWarning') }}
       </Block>
+      <Block
+        v-else-if="
+          space.filters?.minScore > 0 &&
```

Comment:
Can you create a computed var here, it would be better if we reduce the logic in the template